src/ontology/imports/foodon_product_import.owl

```diff
@@ -33599,6 +33605,109 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+        <rdfs:label xml:lang="en">diezmillo</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004168 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004168">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03317170"/>
+        <obo:IAO_0000115 xml:lang="en">The process of slow cooking meat to make it easier to chew or cut.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wikipedia.org/wiki/Meat_tenderness</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-02-04T22:11:53Z</dc:date>
+        <rdfs:label xml:lang="en">meat (tenderized)</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004169 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004169">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001180"/>
+        <obo:IAO_0000115 xml:lang="en">The chicken wings which have been prepared.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-04T22:15:34Z</dc:date>
+        <rdfs:label xml:lang="en">party wings</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004170 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004170">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03309997"/>
+        <obo:IAO_0000115 xml:lang="en">Any foodstuff used to feed livestock.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wiktionary.org/wiki/feedingstuff</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-05T10:24:22Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">feeding stuff</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">feed stuff</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004171 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004171">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003357"/>
+        <obo:IAO_0000115 xml:lang="en">A specially formulated soft and small portion treat to meet the needs of an animal.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-05T11:00:23Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">mini-bite treat</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">petite treat</rdfs:label>
```

Comment:
I lookup "petite treat" on the web and find references to small pastries etc.  So I think "small pet treat" would be a better label (relative to size of animal eating it of course).  I think petite treat is a brand name. But if you want, make it a narrow synonym, and same with "mini-bite treat".  And can we make the definitions as Aristotelian as possible? So, a pet treat which is small in size".  Treats themselves may not be designed to meet [nutritional] needs of an animal, though pleasure is at play!

Reply:
Totally agree. Few brands mentioned the use of these small-bite treats for supplemental feeding, so got confused with that. It's better to change the label and add it as a narrow synonym instead.